### ISSUE TITLE
Support Python 3.10+

### ISSUE THREAD

**Issue by albertyw** [thriftrw/thriftrw-python#166]
The versions of thriftrw-python uploaded to PyPI do not contain any built with new enough releases of Cython to support Python 3.10+.  There should be a new release of thriftrw-python to support Python 3.10+.

**Comment by albertyw** [comment]
I have a set of commits that I plan to convert into pull requests to fix this issue:

```
Yep, I realized that after I created this commit.  My full git history that I plan to convert into pull requests is

```
* 43044c0 - (HEAD -> release, origin/release) Release 1.9.0 (8 days ago) <Albert Wang>
*   b42ae38 - (origin/actions, actions) Merge branch 'dep-update' into actions (8 days ago) <Albert Wang>
|\
| * 4391825 - (origin/dep-update, dep-update) Fix pytest issues (8 days ago) <Albert Wang>
| * b0bb2e3 - Fix outdated setup.cfg (8 days ago) <Albert Wang>
| * aebd214 - Update pytest-benchmark to 4.0.0 (8 days ago) <Albert Wang>
| * 4f7417e - Update pytest-cov package to 4.0.0 (8 days ago) <Albert Wang>
| * dacf6f7 - Update pytest package to 7.2.1 (8 days ago) <Albert Wang>
| * 3c5cfe3 - (origin/python-3.10, python-3.10) Require a newer version of cython to support python 3.10+ (12 days ago) <Albert Wang>
* | 7caae96 - Convert from travis to github actions (8 days ago) <Albert Wang>
* | bb747c0 - (origin/update-python, update-python) Update python version support and tests (8 days ago) <Albert Wang>
* | 7aa66f3 - (origin/remove-sudo, remove-sudo) Remove custom travis configs (8 days ago) <Albert Wang>
|/
* 2595cde - (origin/master, origin/HEAD, fork/master, master) README: Add archival notice (9 months ago) <Abhinav Gupta>
```
```

**Comment by albertyw** [comment]
Python 3.10+ support landed and released as [thriftrw 1.9.0](https://pypi.org/project/thriftrw/1.9.0/)
